webApi/webApi/Controller/StudentController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Newtonsoft.Json;
+using StudentData;
+using StudentRepositry;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace webApi.Controller
```

Comment:
Namespace name is small

Reply:
Sir now i can not change because it will generate a lot of  error and next i will not repeate this mistake

Reply:
i also changed this